resource/copilot/SSS_约翰老妈在建地块_浊蒂水陈版.json

```diff
@@ -39,7 +39,7 @@
         {
             "name": "假日威龙陈",
             "skill": 3,
-            "skill_usage": 0
+            "skill_usage": 1
```

Comment:
@Manicsteiner I'll merge this before https://github.com/MaaAssistantArknights/MaaAssistantArknights/pull/7346 because I've made some heavy changes to the copilot json. I've already transferred these new changes (as I also noticed Ch'En alter wasn't working xD)